tests/scalers/arangodb/helper.go

```diff
@@ -0,0 +1,116 @@
+//go:build e2e
+// +build e2e
+
+package arangodb
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/kedacore/keda/v2/tests/helper"
+)
+
+type templateData struct {
+	Namespace  string
+	Database   string
+	Collection string
+}
+
+const arangoDeploymentTemplate = `apiVersion: "database.arangodb.com/v1"
+kind: "ArangoDeployment"
+metadata:
+  name: "example-arangodb-cluster"
+  namespace: {{.Namespace}}
+spec:
+  architectures:
+    - arm64
+    - amd64
+  mode: Cluster
+  image: "arangodb/arangodb:3.10.1"
+`
+
+func InstallArangoDB(t *testing.T, kc *kubernetes.Clientset, testNamespace string) {
+	t.Log("installing arangodb crds")
+	_, err := helper.ExecuteCommand(fmt.Sprintf("helm install arangodb-crds https://github.com/arangodb/kube-arangodb/releases/download/1.2.20/kube-arangodb-crd-1.2.20.tgz --namespace=%s --wait", testNamespace))
+	assert.NoErrorf(t, err, "cannot install crds - %s", err)
+
+	t.Log("installing arangodb operator")
+	_, err = helper.ExecuteCommand(fmt.Sprintf("helm install arangodb https://github.com/arangodb/kube-arangodb/releases/download/1.2.20/kube-arangodb-1.2.20.tgz --set 'operator.architectures={arm64,amd64}' --namespace=%s --wait", testNamespace))
+	assert.NoErrorf(t, err, "cannot create operator deployment - %s", err)
+
+	t.Log("creating arangodeployment resource")
+	helper.KubectlApplyWithTemplate(t, templateData{Namespace: testNamespace}, "arangoDeploymentTemplate", arangoDeploymentTemplate)
+	assert.True(t, helper.WaitForPodCountInNamespace(t, kc, testNamespace, 11, 5, 20), "pod count should be 11")
+	assert.True(t, helper.WaitForAllPodRunningInNamespace(t, kc, testNamespace, 5, 20), "all pods should be running")
```

Comment:
What is this for? I guess it's to ensure the cluster is ready, but maybe we can monitor the deployment or the statefulset if arango doesn't use a custom controller (I have no idea about it tbh)

Reply:
You're right, however, it gets created as a part of a CR. It looks something like this: 
```
❯ kg arangodeployments.database.arangodb.com

NAME                                      AGE
example-arangodb-cluster   148m
```

Hence using the pod status was the most straightforward way to check.